Gopkg.toml

```diff
@@ -17,7 +17,7 @@ required = [
 
 [[override]]
   name = "github.com/gardener/gardener-resource-manager"
-  revision = "0.3.0"
+  revision = "0.4.0"
```

Comment:
Because of https://github.com/gardener/gardener-resource-manager/pull/6 another version / commit must be referred to, right?